.circleci/config.yml

```diff
@@ -99,7 +99,7 @@ commands:
           key: dependency-cache-{{ checksum "package.json" }}
       - run:
           name: Installing dependencies
-          command: npm install
+          command: npm install --legacy-peer-deps
```

Comment:
The latest docker images are using latest node which is causing some errors due to how latest npm handles peer dependencies. This flag is saying, don't do something new. Check this page for more [details](https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh). To fix this, we need to update all our dependencies and our codebase to make it work with latest node/npm. Not sure how that'll work because some dependencies have specific version requirements. So this requires more effort to upgrade to latest versions. Btw, see our thread with Manjesh [here](https://twilio.slack.com/archives/C0ANJHZLN/p1623366716139500?thread_ts=1623366515.139400&cid=C0ANJHZLN) for background.